tests/torch/quantization/test_strip.py

```diff
@@ -365,57 +356,6 @@ def check_compression_modules(
     assert isinstance(compression_module, expected_class)
 
 
-@pytest.mark.parametrize(
-    ("mode", "decompressor_class", "torch_dtype", "atol"),
-    (
-        (CompressWeightsMode.INT4_ASYM, INT4AsymDQ, torch.float32, 1e-3),
-        (CompressWeightsMode.INT4_ASYM, INT4AsymDQ, torch.float16, 1e-8),
-        (CompressWeightsMode.INT4_ASYM, INT4AsymDQ, torch.bfloat16, 1e-2),
-        (CompressWeightsMode.INT4_SYM, INT4SymDQ, torch.float32, 1e-3),
-        (CompressWeightsMode.INT4_SYM, INT4SymDQ, torch.float16, 1e-8),  # torch.compile introduces bigger diff for sym
-        (CompressWeightsMode.INT4_SYM, INT4SymDQ, torch.bfloat16, 1e-2),
-        (CompressWeightsMode.INT8_SYM, INT8SymDQ, torch.bfloat16, 1e-8),  # int8 uses per-channel vs int4 group-wise
-        (CompressWeightsMode.INT8_ASYM, INT8AsymDQ, torch.bfloat16, 1e-8),  # int8 uses per-channel vs int4 group-wise
-    ),
-)
-def test_nncf_strip_lora_model(mode, decompressor_class, torch_dtype, atol, mocker):
-    input_shape = [1, 16]
```

Comment:
Is it safe to remove this test? it's quite recent one

Reply:
https://github.com/openvinotoolkit/nncf/blob/13803dbe247dc126c9760f1293299990cabba0f2/tests/torch2/function_hook/quantization/strip/test_strip_dequantize.py#L93